src/DetectorMap.cc

```diff
@@ -379,7 +367,7 @@ DetectorMap::_set_xToFiberId()
     float lastCenter = -1e30;            // center of the last fibre we passed scanning to the right
     std::size_t lastIndex = -1;          // index of the last fibre we passed.  Special case starting with 0
 
-    float nextCenter = getXCenter(_fiberIds[1], iy); // center of the next fiber we're looking at
```

Comment:
Are you sure about this?  fiberIds start at 1, not 0.

Reply:
This isn't about the fiberIds, but the indices of the vector, which start at zero.

Reply:
I know that, but I was concerned that I might have adjusted the size of that array.  If you've checked that I didn't, this is fine.

Reply:
I just double-checked, and I don't see any size adjustment or sneaky indexing going on. Perhaps there was, once upon a time (?), but there isn't now.